src/gr11review/part2/UtilityTest.java

```diff
@@ -2,9 +2,284 @@
 
 import static org.junit.jupiter.api.Assertions.*;
 import org.junit.jupiter.api.Test;
-
-import gr11review.part2.Utility;
+import java.io.IOException;
+import java.util.Arrays;
 
 public class UtilityTest{
+    // Methods1 Test1: Testing for true xybalance
+    @Test
+    public void xyBalanceTest1() {
+        assertEquals(true, Utility.xyBalance("aaxbby"));
+    }
+
+    // Methods1 Test2: Testing for only x
+    @Test
+    public void xyBalanceTest2() {
+        assertEquals(false, Utility.xyBalance("aaxbb"));
+    }
+
+    // Methods1 Test3: Testing for y before x
+    @Test
+    public void xyBalanceTest3() {
+        assertEquals(false, Utility.xyBalance("yaaxbb"));
+    }
+
+    // Methods1 Test4: Testing for empty string
+    @Test
+    public void xyBalanceTest4() {
+        assertEquals(false, Utility.xyBalance(" "));
+    }
+
+    // Methods1 Test5: Testing for number string (no x's and no y's)
+    @Test
+    public void xyBalanceTest5() {
+        assertEquals(false, Utility.xyBalance("12345"));
+    }
+    
+    // Methods1 Test6: Testing for only y
+    @Test
+    public void xyBalanceTest6() {
+        assertEquals(false, Utility.xyBalance("abbby"));
+    }
   
+
+    
+    // Methods2 Test1: Testing README Example 1
+    @Test
+    public void sumNumbersTest1() {
+        assertEquals(123, Utility.sumNumbers("abc123xyz"));
+    }
+
+    // Methods2 Test2: Testing README Example 2
+    @Test
+    public void sumNumbersTest2() {
+        assertEquals(44, Utility.sumNumbers("aa11b33"));
+    }
+
+    // Methods2 Test3: Testing README Example 3
+    @Test
+    public void sumNumbersTest3() {
+        assertEquals(18, Utility.sumNumbers("7 11"));
+    }
+
+    // Methods2 Test4: Testing empty string
+    @Test
+    public void sumNumbersTest4() {
+        assertEquals(0, Utility.sumNumbers(""));
+    }
+
+    // Methods2 Test5: Testing long string
+    @Test
+    public void sumNumbersTest5() {
+        assertEquals(214119, Utility.sumNumbers("fasdl213784ds234oqw93k8"));
+    }
+
+
+
+    // FileIO1 Test1: Testing words.txt
+    @Test
+    public void fileIO1Test1() throws IOException {
+        assertEquals("consectetur", Utility.longestWord("words.txt")); // FILE NOT FOUND 
+    }
+
+
+
+    // Array1 Test1: Tests to see if the output removed the two 10s
+    @Test
+    public void withoutTenTest1() { 
+        int[] numsInput = {1, 10, 10, 2};
+        int[] numsExpected = {1, 2, 0, 0};
+        assertEquals(numsExpected, Utility.withoutTen(numsInput));
```

Comment:
use `assertArrayEquals` instead to check to see if values in two arrays are a match.  using `assertEquals` in this case compare the memory address.  Not what you want.